### PR TITLE
Fix wrong inferface name in comment

### PR DESCRIPTION
**Pull request by hong9lol**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
change correct interface name in the comment.

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/server/stats/handler.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ type Provider interface {
 	//
 	// ListPodStats returns the stats of all the containers managed by pods.
 	ListPodStats(ctx context.Context) ([]statsapi.PodStats, error)
-	// ListPodStatsAndUpdateCPUNanoCoreUsage updates the cpu nano core usage for
+	// ListPodCPUAndMemoryStats updates the cpu nano core usage for
```
Comment (yujuhong) [inline pkg/kubelet/server/stats/handler.go:44]: `updates the cpu nano core usage for` --> is this still true?

**Comment by hong9lol** [comment]
@dims @krmayankk Could someone help review this PR?

**Pull request closed without merging**